auth/jwt_test.go

```diff
@@ -13,6 +13,8 @@ import (
 
 type JWTAuthenticatorTestSuite struct {
 	goyave.TestSuite
+	tokenAuthenticator *JWTAuthenticator
+	user               *TestUser
```

Comment:
We can afford duplication in tests, it's not a problem. I would prefer to re-define those in all tests in case we need different parameters. (Like in TestOptional) This also ensures with don't have tests overflowing on others.